docs/notebooks/mister_p.ipynb

```diff
@@ -0,0 +1,8218 @@
+{
```

Comment:
<p>could you install <code>IProgress</code> so this looks nicer? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/716/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>I tried that and it didn't work so i just suppressed the output using %%capture instead...</p>